src/utils/cartitem-actions.js

```diff
@@ -0,0 +1,89 @@
+import axios from "axios";
+
+const addToCart = async (product, dispatchCart,navigate) => {
+  let response;
```

Comment:
this can be a const and inside try block

Reply:
made changes